xmind/tests/test_workbook_element.py

```diff
@@ -0,0 +1,751 @@
+import logging
```

Comment:
Overall concern for patching via ```_init_patch_with_name```
It's more safe to pass ```auto_spec=True``` during patching the method and then check that the method has been called with an object on which we expect a call as a first parameter during ```patched_method.assert_called_once_with(**OBJ**, param1, param2)```